packer/ansible/roles/images/tasks/main.yml

```diff
@@ -20,3 +20,11 @@
    - monorail-efi32-snponly.efi
    - monorail-efi64-snponly.efi
    - undionly.kpxe
```

Comment:
I see the undionly.kkpxe is fetched from line 30 (from syslinux). ipx build will only generate undionly.kpxe, I think keep the undionly.kpxe is intentionally for future use. Of course, remove this right now will not impact current function in verified platforms.